readthedocs/projects/views/private.py

```diff
@@ -111,38 +111,53 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ProjectUpdate(ProjectSpamMixin, PrivateViewMixin, UpdateView):
+class ProjectMixin(PrivateViewMixin):
```

Comment:
Just with the name of the class, I'm not 100% sure if all the Project views need to inherit from this class or not. I guess, yes. Although, it may be good to reflect what the mixin is about in the name of the class, or if that's tricky, in a docstring.

Reply:
All of them inherit this mixin but ProjectDasboard, because it's very different, it doesn't share anything in common.